package.json

```diff
@@ -27,5 +27,6 @@
   },
   "scripts": {
     "test": "grunt test"
-  }
+  },
+  "main": "src/countdown.js"
```

Comment:
Either seems fine! I prefer `src` in case someone forgot to run the build script, but then again if that happens, you've got bigger issues ;-) I'll change it to `dist`